### PR TITLE
Updating usage panel and other panel headers

### PR DESCRIPTION
**Pull request by line47**
Updating the semantics of the headers used in panels

### CODE REVIEW

File: static_src/components/app_container.jsx [hunk 1]
```diff
@@ -140,6 +140,7 @@ export default class AppContainer extends React.Component {
           <h2>{ this.fullTitle } { this.statusUI }</h2>
           { this.restart }
           <Panel title="Usage and allocation">
+              <span>View more usage data at <a href="https:logs.cloud.gov">logs.cloud.gov</a></span>
```
Comment (adborden) [inline static_src/components/app_container.jsx]: You're missing the `//` in the protocol, no?

**Comment by adborden** [comment]
Are there cg-style changes that are paired with this PR?

**Comment by line47** [comment]
No cg-style updates to pair with for this PR.

**Comment by line47** [comment]
After looking over at cg-style it looks like this is paired with https://github.com/18F/cg-style/pull/229/

**Comment by adborden** [comment]
:sunglasses: that looks much better. In that case, I'd like to call out that I don't think we should be using CSS rules based on elements, but rather create new class-based rules instead. I believe @msecret and @thisisdano are in agreement?

File: static_src/components/app_container.jsx (hunk 1 above)
Comment (line47) [inline static_src/components/app_container.jsx]: Good catch, I've added the `//`

**Comment by line47** [comment]
pairs with https://github.com/18F/cg-style/pull/238

**Pull request merged**